packages/core-typings/src/omnichannel/routing.ts

```diff
@@ -12,7 +12,8 @@ export type RoutingMethodConfig = {
 	autoAssignAgent: boolean;
 };
 
-export type SelectedAgent = { agentId: string; username: string };
+export type SelectedAgent = { agentId?: string; username?: string };
```

Comment:
is that possible? having an agent with no id or username?

Reply:
It's "technically" possible, as some places return `{}` or `null` as the agent. But for the case it matters here, at least `agentId` is required and is present always, so we should be good by making it required 👀 

Good one